### PR TITLE
Update duckduckgo.po

### PR DESCRIPTION
**Pull request by omeritzics**

### CODE REVIEW

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -96,17 +100,19 @@ msgstr ""
 
 msgctxt "noresults"
 msgid ""
-"%sClick here%s to try again, if you think there should be results for this "
+"%sClick here%s to try again, if you think there should be results for this"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:103]:
```suggestion
"%sClick here%s to try again, if you think there should be results for this "
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -77,15 +79,17 @@ msgstr ".בנו %s"
...
 #. The placeholder here will be a bold number. Sorry, this is hard to translate.
 msgctxt "homepage ATB social proof"
 msgid "%s%s%s reviews"
 msgstr ""
 
 msgctxt "install-duckduckgo"
-msgid "%sClick %sAllow%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
+msgid ""
+"%sClick %sContinue To Installation%sClick %sAdd%sCheck %sAllow%s, then click "
+"%sOkay%s"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:92]: Please, do not change the original strings. Original strings are tagged `msgid`, while the translations are `msgstr`.

```suggestion
msgid "%sClick %sAllow%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -99,14 +101,16 @@ msgid ""
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
 msgstr ""
+"%sלחץ כאן%s כדי לנסות שוב, if you think there should be results for this "
+"search."
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:105]: This is the string I wanted you to double check. As you can see, it is only partially translated.

**Comment by GioSensation** [comment]
Hi @omeritzics. Thanks for looking into this again, much appreciated. I have fixed a couple of issues and highlighted again the string that is only half-translated. Would you mind taking another look at it?

A few notes for you in the future:

- Do not change the original strings (the ones tagged `msgid`). Only operate on the strings tagged `msgstr`.
- Tools such as [Poedit](https://poedit.net) will help you a lot with the translations by alerting you if you are introducing formatting inconsistencies. They will also give you a cleaner UI for you to work with, so that you avoid changing original strings. I highly recommend using a specialized tool. I use Poedit myself, but there are many others.
- When you are asked to review a Pull Request as I am doing here, do not open a new one. Keep working on the same PR, otherwise the reviewer must go through the whole thing again. If you keep working on the same PR, I can easily check the new commits and merge the PR.

Thanks again for your contribution. I look forward to merging this. 💪

**Comment by GioSensation** [comment]
@omeritzics This PR hasn't been updated for quite a while. I will mark it stale and it will be closed in the next few days. If you need more time or have questions, please ping me here.

**Comment by omeritzics** [comment]
Hello, I have been opened a new PR with the updated files. Link: #376

**Pull request closed without merging**